_source/_posts/2023-07-27-spring-graphql-react.md

```diff
@@ -0,0 +1,1837 @@
+---
+layout: blog_post
+title: "How to build a GraphQL API with Spring for GraphQL"
```

Comment:
This ranks well (69) on our title analyzer. However, Spring Boot has more searches:

- spring graphql: 2400/month
- spring boot graphql: 3600/month

For this reason, I suggest we change it to "How to Build a GraphQL API with Spring Boot." This gets a 68, which is still pretty good.